libraries/AP_SmartRTL/AP_SmartRTL.h

```diff
@@ -74,6 +75,12 @@ class AP_SmartRTL {
     // run background cleanup - should be run regularly from the IO thread
     void run_background_cleanup();
 
+    // returns true if pilot's yaw input should be used to adjust vehicle's heading
+    bool use_pilot_yaw(void) const;
+
+    // returns true if option bit is set to give back yaw control after no user input
+    bool give_back_yaw_ctrl(uint32_t time_since_last_input) const;
```

Comment:
maybe call the method, "restore_auto_yaw_after_5sec()" or something similar.  Might as well be specific.